ci/tasks/deploy.yml

```diff
@@ -0,0 +1,19 @@
+---
+platform: linux
+
+image_resource:
+  type: registry-image
+  source:
+    repository: cybojenix/govwifi-concourse-runner
+
+inputs:
+  - name: govwifi-admin
+
+params:
+  DEPLOY_STAGE:
+  AWS_ACCESS_KEY_ID:
+  AWS_SECRET_ACCESS_KEY:
+  AWS_DEFAULT_REGION:
```

Comment:
Does this make the params required?

Reply:
I believe they'll default to a blank string. The docs on this are a bit vague